README.md

```diff
@@ -130,3 +130,92 @@ example_dat %>%
     ## 7 happy        Yes    138 (57.26%) 137 (59.57%)    0.855 
     ## 8 sugar_factor ""     0.46 (0.3)   0.52 (0.29)     0.0129
     ## 9 age          ""     42 [22, 62]  41 [26, 61]     0.888
+
+#### `format_tbl`
+
+``` r
+univariate <- example_dat %>% 
+  descriptives(
+    variables = c("age", "sugar_factor", "gender", "happiness", "happy")
+  )
+
+univariate
+```
+
+    ## # A tibble: 9 x 3
+    ##   Variable     Label  Statistics  
+    ##   <chr>        <chr>  <chr>       
+    ## 1 gender       female 235 (46.91%)
+    ## 2 gender       male   266 (53.09%)
+    ## 3 happiness    happy  367 (77.75%)
+    ## 4 happiness    sad    105 (22.25%)
+    ## 5 happy        no     105 (22.25%)
+    ## 6 happy        yes    91 (19.28%) 
+    ## 7 happy        Yes    276 (58.47%)
+    ## 8 age          ""     42.3 (22.1) 
+    ## 9 sugar_factor ""     0.49 (0.3)
+
+``` r
+univariate %>%
+  format_tbl()
+```
+
+    ## # A tibble: 12 x 3
+    ##    Variable     Label  Statistics  
```

Comment:
Should the univariate table also have the sample size appended to the `Statistics` column name?

Reply:
It can if we want it to. In practice, I only add N's to the treatment levels.

Reply:
I think it's worth it to add that in for the summary of the sample to be complete.